integration_tests/test_quick.py

```diff
@@ -763,6 +766,72 @@ def test_repay_and_delay_interest(self):
         self.assertEqual(len(txs), 1)
         self.assertEqual(txs[0]["amount"], 13_000)      
         
+    def test_withraw_admin_rewards(self):
+        chain = LocalChain(storage=self.storage)
+
+        self.add_token(chain, token_a)
+        self.add_token(chain, token_b)
+
+        chain.execute(self.ct.mint(0, 40_000))
+        chain.execute(self.ct.enterMarket(0))
+        
+        chain.execute(self.ct.borrow(1, 10_000))
+
+        chain.advance_blocks(1)
+
+        self.update_price_and_interest(chain, 0, 100, one_percent_per_second)
+        self.update_price_and_interest(chain, 1, 100, one_percent_per_second)
+
+        chain.execute(self.ct.repay(1, 13_000))
+        res = chain.execute(self.ct.withdrawReserve(1, 1_500), sender=admin)
+        transfers = parse_transfers(res)
+        self.assertEqual(transfers[0]["amount"], 1_500)
+
+        # nothing left to repay and withdraw
+        with self.assertRaises(MichelsonRuntimeError):
+            chain.execute(self.ct.repay(1, 1))
+            chain.execute(self.ct.withdrawReserve(1, 1), sender=admin)
```

Comment:
I dont think this action is meaningful since exeption is ought to be raised on `repay`

Reply:
Please move it to a separate `with self.assertRaises`